apps/lib/p5play/p5.play.js

```diff
@@ -1558,9 +1552,12 @@ function Sprite(pInst, _x, _y, _w, _h) {
       imageMode(CENTER);
 
       translate(this.position.x, this.position.y);
+
       // This line differs from p5.play upstream because GameLab users
       // should be able to change the width and height of their animations.
-      scale(this.scale*dirX*this._horizontalStretch, this.scale*dirY*this._verticalStretch);
+      var scaleX = this.modifyScaleX();
+      var scaleY = this.modifyScaleY();
+      scale(scaleX*dirX, scaleY*dirY);
```

Comment:
I don't love the names for `modifyScaleX`/`Y`, which are named as verbs but in practice are getters with no side effects.  They should also probably be private-named (leading underscore) and have a descriptive comment.

More generally, I wonder if we want to create just one method that we'll override, something like `this._applySpriteScaleForRendering`.  I can't think of a situation where we'd want to override the horizontal scale factor but not the vertical one; and even if we did, it'd be pretty simple to do so with the single override method.